src/invoices/InvoicesList/Invoice/InvoiceDetailsLayer.js

```diff
@@ -18,6 +20,14 @@ class InvoiceDetailsLayer extends Component {
       ...invoiceLinesResource,
       fetch: false,
     },
+    vendor: {
```

Comment:
it's strange, but previously I commented this as `copy paste`, but comment disappeared